config/resources/crd/kustomization.yaml

```diff
@@ -3,6 +3,7 @@
 # It should be run by config/default
 commonLabels:
   cluster.x-k8s.io/v1alpha3: v1alpha3
+  cluster.x-k8s.io/v1alpha4: v1alpha4
```

Comment:
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/. Old versions shouldn't be deleted. Instead, there should be one stored version and all other version should be converted to that version. In case of controller-runtime, hub and spokes model for conversion is used: https://book.kubebuilder.io/multiversion-tutorial/conversion.html. That's needed for smooth migration of users to newer versions of controller.